alf/algorithms/algorithm.py

```diff
@@ -163,13 +163,15 @@ def __init__(self,
         # See the ``force_params_visible_to_parent`` property below for details.
         self._force_params_visible_to_parent = False
         self._grad_scaler = None
+        self._temporally_independent_train_step = None
         if config:
```

Comment:
config cannot be None because we use `self._config` everywhere in the file?

Reply:
config is not always passed to the Algorithm constructor.

Reply:
> config is not always passed to the Algorithm constructor.

Then some places will cause errors? Because they don't check if `self._config` is None

Reply:
Yes, that will cause error. But many algorithm will not go to those places.